backend/inspirehep/assign/views.py

```diff
@@ -84,16 +79,23 @@ def assign_to_new_stub_author(author_record, literature_recids):
 def assign_to_author(from_author_recid, to_author_recid, literature_recids):
     author_record = AuthorsRecord.get_record_by_pid_value(to_author_recid)
     num_workers = count_consumers_for_queue("assign")
+    assignment_tasks = []
     for batch in chunker(literature_recids, 10, num_workers):
-        current_celery_app.send_task(
+        task_signature = current_celery_app.signature(
             "inspirehep.assign.tasks.assign_papers",
             kwargs={
                 "from_author_recid": from_author_recid,
                 "to_author_record": author_record,
```

Comment:
wait are we passing the whole author record around? 😵‍💫 could we please refactor that (or add a new task)